Gemfile

```diff
@@ -63,5 +63,5 @@ group :production do
 end
 
 gem 'bootstrap-sass', '~> 3.1.1'
-gem 'devise'
-gem 'airbrake'
\ No newline at end of file
+#gem 'devise'
```

Comment:
removed Devise to allow for creation of User model. Devise create User model, controller, route, etc. by default on installation. I removed here to allow for creation of User model